File: Arrays/Easy/Python/3289. The Two Sneaky Numbers of Digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:

        s = set()
        arr = []

        for i in nums:
            if i not in s:
                s.add(i)
            else:
                arr.append(i)
        
        return arr
        

File: Arrays/Easy/3289. The Two Sneaky Numbers of Digitville.java
class Solution {
    public int[] getSneakyNumbers(int[] nums) {

        // Time Complexity : O(n)

        // Space Complexity : O(n)
        
        
        Set<Integer> set = new HashSet<>();
        int[] ans = new int[2];
        int c = 0;

        for(Integer i: nums){
            if(!set.contains(i)){
                set.add(i);
            }
            else{
                ans[c] = i;
                c++;

                if(c >= 2){
                    break;
                }
            }
        }

        return ans;
    }
}
